src/GetComment.js

```diff
@@ -0,0 +1,15 @@
+import { BASE_URL } from './apis.js';
+
+const getComment = async (itemId) => fetch(`${BASE_URL}/comments?item_id=${itemId}`)
+  .then((response) => {
+    if (response.ok) {
+      return response.json();
+    }
+    throw new Error(`Error: ${response.status}`);
+  })
+  .then((data) => data);
+  // .catch((error) => {
+  //   console.error(error);
+  // });
+
+export default getComment;
```

Comment:
- Please note that when a comment is clicked, the screen is not updated until the page is refreshed. Kindly make sure a user clicks on the "Comment" button, the data is expected to get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen updated. 👍🏽

Reply:
Well resolved 🚀 . An added comment is added instantly without the need to refresh the page. Well done!!! 👏🏾